addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -16,6 +16,17 @@ options.registry.gallery = options.Class.extend({
      */
     start: function () {
         var self = this;
+        // TODO In master: define distinct classes.
+        // Differentiate both instances of this class.
```

Comment:
I would be a bit more explicit here. Add that we don't want to register the event listeners twice :+1: